### PR TITLE
FAI-842 - add PDP example notebook

### PR DESCRIPTION
**Pull request by tteofili**
Added example notebook demonstrating how to use Partial Dependence Plot explainer in TrustyAI.

### CODE REVIEW

File: examples/PDP.ipynb [hunk 1]
```diff
@@ -0,0 +1,679 @@
...
+   "id": "e4c8a080",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "SLF4J: Failed to load class \"org.slf4j.impl.StaticLoggerBinder\".\n",
+      "SLF4J: Defaulting to no-operation (NOP) logger implementation\n",
+      "SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.\n"
```
Comment (danielezonca) [inline examples/PDP.ipynb:15]: We need some followup ticket to provide a default SLF4J conf to avoid this error

File: examples/PDP.ipynb [hunk 2]
```diff
@@ -0,0 +1,679 @@
...
+    "from trustyai.utils import TestModels\n",
+    "from trustyai.explainers.pdp import PDPExplainer, PredictionProviderMetadata\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.datasets import make_classification"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (danielezonca) [inline examples/PDP.ipynb:30]: Can you add some text/description of each (or sort of) step? Just to briefly explain what is going on

Comment (tteofili) [reply examples/PDP.ipynb:30]: sure, will do.

File: examples/PDP.ipynb (hunk 1 above)
Comment (tteofili) [reply examples/PDP.ipynb:15]: good point: https://issues.redhat.com/browse/FAI-912

**Pull request merged**